packages/react-components/stories/emoji-variations-list.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from 'react'
+import fetchMock from 'fetch-mock'
+import { jsxDecorator } from 'storybook-addon-jsx'
+import { boolean, color, number, withKnobs } from '@storybook/addon-knobs'
+import { Story } from '@storybook/react'
+import { GiphyFetch } from '@giphy/js-fetch-api'
+import type { GifID, IGif } from '@giphy/js-types'
+
+import inTestsRunner from './in-tests-runner'
+import mockDefaultEmojiVariationsResult from './mock-data/default-emoji-variations.json'
+import mockEmojiVariationsResult from './mock-data/emoji-variations.json'
+import { EmojiVariationsList, EmojiVariationsListProps, Grid } from '../src'
+
+const apiKey = 'sXpGFDGZs0Dv1mmNFvYaGUvYwKX0PWIh'
+const gf = new GiphyFetch(apiKey)
+
+export default {
+    title: 'React Components/Emoji Variations List',
+    decorators: [withKnobs, jsxDecorator],
+}
+
+type StoryProps = Partial<EmojiVariationsListProps>
```

Comment:
@pshoniuk optionally instead of exporting a component's props, you can just use the component and `ComponentProps<typeof EmojiVariationsList>`

Reply:
@giannif I exported them just in case a user prefers explicit types. 
Since ComponentProps is a bit heavy in large projects, it might be preferable to use explicit types to optimize TS build time and code autocomplete.